api.planx.uk/server.ts

```diff
@@ -471,6 +472,8 @@ app.get("/gis", (_req, res, next) => {
 
 app.get("/gis/:localAuthority", locationSearch);
 
+app.get("/roads", classifiedRoadsSearch);
```

Comment:
nit: I think this could be a bit more meaningful with a route like `/gis/roads` or `/gis/road-classification`

Reply:
Ah yeah that's a vary valid point - `/gis/roads` would match `/gis/:localAuthority`. Let's revisit this in the future at some point 👍